build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```

Comment:
To use version `4.0` the users need to have Android Studio 3.0+ which is still a beta release, most of the users stick to a stable release - current stable release 2.3.3, and Android Studio below 3.x can't use the new gradle versions (links: [Android dev](https://developer.android.com/studio/preview/features/new-android-plugin-migration.html) and [SO](https://stackoverflow.com/questions/44580320/how-to-use-gradle-4-0-in-android-studio-2-3)). So, I think its better to stick with `2.3.3` till Android Studio 3.0 becomes a stable release :smile: .